client/src/Components/Stories/index.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react';
+
+class Stories extends Component {
+	state = {};
+	render() {
+		return <h1>this is a stories</h1>;
+	}
+}
```

Comment:
This seems like a great opportunity for Stateless Functional Component (SFC) 😄

Reply:
this section related to my colleague don't worry about it